Upstream-Status: Pending [Not submitted to upstream yet]
Signed-off-by: Abdellatif El Khlifi <abdellatif.elkhlifi@arm.com>

From 3ee38ef07bd82c843497dc4e69a4d4c5f21dbbf7 Mon Sep 17 00:00:00 2001
From: Vishnu Banavath <vishnu.banavath@arm.com>
Date: Tue, 26 Oct 2021 18:29:05 +0100
Subject: [PATCH] arm: corstone1000: amend kernel bootargs with ip=dhcp
 earlyprintk

This change is to
* pass ip=dhcp required for ethernet to get and ip
* enable earlyprintk to print kernel logs before the actual
kernel driver comes up

Signed-off-by: Vishnu Banavath <vishnu.banavath@arm.com>

diff --git a/configs/corstone1000_defconfig b/configs/corstone1000_defconfig
index cfe80cf5f4..7574553e83 100644
--- a/configs/corstone1000_defconfig
+++ b/configs/corstone1000_defconfig
@@ -8,7 +8,7 @@ CONFIG_IDENT_STRING=" corstone1000 aarch64 "
 CONFIG_FIT=y
 CONFIG_BOOTDELAY=3
 CONFIG_USE_BOOTARGS=y
-CONFIG_BOOTARGS="console=ttyAMA0 loglevel=9"
+CONFIG_BOOTARGS="console=ttyAMA0 loglevel=9 ip=dhcp earlyprintk"
 # CONFIG_DISPLAY_CPUINFO is not set
 # CONFIG_DISPLAY_BOARDINFO is not set
 CONFIG_HUSH_PARSER=y
-- 
2.17.1

